main.cpp

```diff
@@ -205,6 +460,27 @@ Thing 10) Synth
     3)  Send midi
  */
 
+struct synth
+{
+    //Oscillator
+    double oscillatorAudio = 0.0; 
+    //LFO
+    double controlValueLFO = 0.0; 
+    //Envelope
+    double controlValueEnvelope = 0.0; 
+    //Mixer
+    double mixerAudio = 0.0;
+    //Noise Generator
+    double noiseAudio = 0.0;
```

Comment:
These are supposed to be instances of your UDTs, not primitives. for example:
`Oscillator oscillator;` not `double oscillatorAudio;`
- [ ] revise.